.travis.yml

```diff
@@ -3,6 +3,8 @@ sudo: false
 language: node_js
 
 node_js:
+  - '8'
+  - '7'
   - '6'
   - '5'
-  - '4'
+  - '4'
```

Comment:
We should drop everything till version 8, but since v6 is causing no issues we can keep those for a while